extensions/azure/processors/AzureBlobStorageProcessorBase.h

```diff
@@ -51,10 +51,13 @@ class AzureBlobStorageProcessorBase : public AzureStorageProcessorBase {
       .withDescription("The storage account key. This is an admin-like password providing access to every container in this account. "
           "It is recommended one uses Shared Access Signature (SAS) token instead for fine-grained control with policies.")
       .supportsExpressionLanguage(true)
+      .isSensitive(true)
       .build();
   EXTENSIONAPI static constexpr auto SASToken = core::PropertyDefinitionBuilder<>::createProperty("SAS Token")
       .withDescription("Shared Access Signature token. Specify either SAS Token (recommended) or Storage Account Key together with Storage Account Name if Managed Identity is not used.")
-      .supportsExpressionLanguage(true).build();
+      .supportsExpressionLanguage(true)
```

Comment:
I don't think this supports EL. The EL extension only overrides the getProperty overloads that take a flow file for context, and the `onSchedule` of this class calls the overload without a flow file parameter.

Reply:
`onSchedule` only does a sanity check that `SASToken` is non-empty; in `AzureBlobStorageProcessorBase::getAzureCredentialsFromProperties()` we use `getProperty()` with a flow file argument